Makefile

```diff
@@ -1,7 +1,9 @@
 -include ../../config.mk
 -include ../../tools/kconfig/.config
 
-CC:=$(CROSS_COMPILE)gcc
+OS:=$(shell uname)
+
+CC:=$(CROSS_COMPILE)clang
```

Comment:
Sure. In my project I compile picotcp for Android under x86 and arm which come with `clang` compiler rather than `gcc`. What would be the preferred way off supplying the compiler to use?